devon4ng-mat-layout/index.asciidoc

```diff
@@ -1,19 +1,19 @@
 = Create an Angular application with Angular Material components using devon4ng
 ====
-The purpose of this tutorial is to get a basic understanding of creating layouts using Angular Material in a devon4ng application. We will create an application with a header containing some menu links and a sidenav with some navigation links.
+The purpose of this tutorial is to get a basic understanding of creating layouts using Angular Material in a devon4ng application. You will create an application with a header containing some menu links and a sidenav with some navigation links.
 
 ## Prerequisites
 * devonfw-ide installed
```

Comment:
To do this tutorial in katacoda you do not have to have the IDE installed on your local system.